Create may-ide-updates.md (#3447)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md

---
title: "May IDE updates and fixes"
id: "may-ide-updates"
description: "May 2023 release note: We've ."
sidebar_label: "Update and fixes: IDE"

> sidebar_position: 7

Author: ```suggestion
sidebar_position: 2
```

tags: [May-2023, IDE]
---

To continue improving your [Cloud IDE](https://docs.getdbt.com/docs/cloud/develop-in-the-cloud) development experience, the dbt Labs team continues to work on adding new features, fixing bugs, and increasing reliability ✨.

Stay up-to-date with [IDE-related changes](https://docs.getdbt.com/tags/ide).

## New features 

> - Lint via SQLFluff in beta (reach out if interested, otherwise this will be available for multi-tenant soon!)

Reviewer: let's just say "Lint via SQL Fluff is now available in beta (GA over the next 2-3 weeks)" and remove the comment about reach out

> - Lint via SQLFluff in beta (reach out if interested, otherwise this will be available for multi-tenant soon!)

Author: ```suggestion
- Lint via SQL Fluff is now available in beta (GA over the next 2-3 weeks)
```

- Format markdown files with prettier

> - Leverage developer experience shortcuts, including `Ctrl + `` (toggle history drawer), `CMD + Option + /` (toggle block comment), `CMD + Shift + P` (open command palette), `Option + W` (close editor tab)

Author: ```suggestion
- Leverage developer experience shortcuts, including `Ctrl + /`` (toggle history drawer), `CMD + Option + /` (toggle block comment), `CMD + Shift + P` (open command palette), `Option + W` (close editor tab)
```

> - Leverage developer experience shortcuts, including `Ctrl + `` (toggle history drawer), `CMD + Option + /` (toggle block comment), `CMD + Shift + P` (open command palette), `Option + W` (close editor tab)

Reviewer: the CTRL + ` is funky looking

- Display parent folder name for files with same name in Changes section
- Navigate the new IDE features quickly using [the IDE User Interface](https://docs.getdbt.com/docs/cloud/dbt-cloud-ide/ide-user-interface) help page
- Use `top X` in SQL when previewing in the IDE
- Opt into the new IDE backend layer over the past month (still with dbt-rpc). Ready for beta later in June!

> ## Product refinements 

Author: ```suggestion
## Product refinements 

```

---
title: "May IDE updates and fixes"
id: "may-ide-updates"
description: "May 2023 release note: We've ."
sidebar_label: "Update and fixes: IDE"
sidebar_position: 2
tags: [May-2023, IDE]
---

To continue improving your [Cloud IDE](https://docs.getdbt.com/docs/cloud/develop-in-the-cloud) development experience, the dbt Labs team continues to work on adding new features, fixing bugs, and increasing reliability ✨.

Stay up-to-date with [IDE-related changes](https://docs.getdbt.com/tags/ide).

## New features 
- Lint via SQL Fluff is now available in beta (GA over the next 2-3 weeks)
- Format markdown files with prettier

> - Leverage developer experience shortcuts, including `Ctrl + /`` (toggle history drawer), `CMD + Option + /` (toggle block comment), `CMD + Shift + P` (open command palette), `Option + W` (close editor tab)

Author: ```suggestion
- Leverage developer experience shortcuts, including `Ctrl + \`` (toggle history drawer), `CMD + Option + /` (toggle block comment), `CMD + Shift + P` (open command palette), `Option + W` (close editor tab)
```

- Display parent folder name for files with same name in Changes section
- Navigate the new IDE features quickly using [the IDE User Interface](https://docs.getdbt.com/docs/cloud/dbt-cloud-ide/ide-user-interface) help page
- Use `top X` in SQL when previewing in the IDE
- Opt into the new IDE backend layer over the past month (still with dbt-rpc). Ready for beta later in June!

## Product refinements 

- Performance-related upgrades:
    - Reduced cold start time by 60+%
    - Improved render time of modals in the IDE by 98%
    - Improved IDE performance with dbt Core v1.5+ (faster and snappier – highly encourage you to update your dbt version!)
- Upgraded sqlfmt (which powers the Format button) to 0.18.0
- Updated Build button to change menu options based on file/model type (snapshot, macro, etc.)
- Display message to disable adblocker for file contents error
- Moved Format button to console bar

> - Made many security enhancements in the IDE

Author: ```suggestion
- Made many security enhancements in the IDE
## Bug fixes

- File icon sizes no longer get wonky in small screen
- Toast notifications no longer take over command bar menu
- Hover info inside the text editor no longer gets cut off
- Transition between a file and a recently modified scratchpad no longer triggers a console error
- dbt v1.5+ now can access the IDE
- Confirm button on the Unsaved Changes modal now closes after clicking it
- Long node names no longer overflow in the parsed logs section in history drawer
- Status pill in history drawer no longer scales with longer command
- Tooltip for tab name with a long file name is no longer cut off
- Lint button should no longer available in main branch
```

---
title: "May IDE updates and fixes"
id: "may-ide-updates"

> description: "May 2023 release note: We've ."

Reviewer 2: ```suggestion
description: "May 2023 release note: We've launched SQLFluff in beta, released an IDE UI page, significantly improved IDE performance, improved error messages, fixed bugs, and more."
```

sidebar_label: "Update and fixes: IDE"
sidebar_position: 2
tags: [May-2023, IDE]
---

> To continue improving your [Cloud IDE](/docs/cloud/develop-in-the-cloud) development experience, the dbt Labs team continues to work on adding new features, fixing bugs, and increasing reliability ✨.

Reviewer 2: ```suggestion
To continue improving your [Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) development experience, the dbt Labs team continues to work on adding new features, fixing bugs, and increasing reliability ✨.
```

> To continue improving your [Cloud IDE](https://docs.getdbt.com/docs/cloud/develop-in-the-cloud) development experience, the dbt Labs team continues to work on adding new features, fixing bugs, and increasing reliability ✨.

Reviewer 2: ```suggestion
To continue improving your [Cloud IDE](/docs/cloud/develop-in-the-cloud) development experience, the dbt Labs team continues to work on adding new features, fixing bugs, and increasing reliability ✨.
```

> Stay up-to-date with [IDE-related changes](https://docs.getdbt.com/tags/ide).

Reviewer 2: ```suggestion
Stay up-to-date with [IDE-related changes](/tags/ide).
```

## New features 
- Lint via SQL Fluff is now available in beta (GA over the next 2-3 weeks)
- Format markdown files with prettier

> - Leverage developer experience shortcuts, including ``Ctrl +` `` (toggle history drawer), `CMD + Option + /` (toggle block comment), `CMD + Shift + P` (open command palette), `Option + W` (close editor tab)

Reviewer 2: ```suggestion
- Leverage developer experience shortcuts, including ``Ctrl + ` `` (toggle history drawer), `CMD + Option + /` (toggle block comment), `CMD + Shift + P` (open command palette), `Option + W` (close editor tab)
```

> - Leverage developer experience shortcuts, including `Ctrl + \`` (toggle history drawer), `CMD + Option + /` (toggle block comment), `CMD + Shift + P` (open command palette), `Option + W` (close editor tab)

Author: ```suggestion
- Leverage developer experience shortcuts, including `Ctrl + \`` (toggle history drawer), `CMD + Option + \/` (toggle block comment), `CMD + Shift + P` (open command palette), `Option + W` (close editor tab)
```

> - Leverage developer experience shortcuts, including `Ctrl + \`` (toggle history drawer), `CMD + Option + \/` (toggle block comment), `CMD + Shift + P` (open command palette), `Option + W` (close editor tab)

Author: ```suggestion
- Leverage developer experience shortcuts, including ``Ctrl +` `` (toggle history drawer), `CMD + Option + /` (toggle block comment), `CMD + Shift + P` (open command palette), `Option + W` (close editor tab)
```

- Display parent folder name for files with same name in Changes section

> - Navigate the new IDE features quickly using [the IDE User Interface](https://docs.getdbt.com/docs/cloud/dbt-cloud-ide/ide-user-interface) help page

Reviewer 2: ```suggestion
- Navigate the new IDE features quickly using [the IDE User Interface](/docs/cloud/dbt-cloud-ide/ide-user-interface) help page
```

- Use `top X` in SQL when previewing in the IDE
- Opt into the new IDE backend layer over the past month (still with dbt-rpc). Ready for beta later in June!

## Product refinements 

- Performance-related upgrades:
    - Reduced cold start time by 60+%
    - Improved render time of modals in the IDE by 98%

>     - Improved IDE performance with dbt Core v1.5+ (faster and snappier – highly encourage you to update your dbt version!)

Reviewer 2: sugg linking to upgrade page

```suggestion
    - Improved IDE performance with dbt Core v1.5+ (faster and snappier – highly encourage you to [upgrade your dbt version](/docs/dbt-versions/upgrade-core-in-cloud)!)
```
